etc/config/c++.amazon.properties

```diff
@@ -2051,6 +2051,17 @@ libs.lexy.versions=trunk
 libs.lexy.versions.trunk.version=trunk
 libs.lexy.versions.trunk.path=/opt/compiler-explorer/libs/lexy/trunk/include
 
+libs.libassert.name=Libassert
+libs.libassert.liblink=assert
+libs.libassert.url=https://github.com/jeremy-rifkin/libassert
+libs.libassert.versions=10:11
+libs.libassert.versions.10.version=1.0
+libs.libassert.versions.10.path=/opt/compiler-explorer/libs/libassert/v1.0/include
+libs.libassert.versions.10.libpath=/opt/compiler-explorer/libs/libassert/v1.0
```

Comment:
you have to leave out libpath, it will be automatically handled by our conan implementation